whichone/static/js/game.js

```diff
@@ -477,3 +342,42 @@ function updateLives() {
     icon3.src = "/static/resources/spotify-icon-dark.png";
   }
 }
+
+function calculateScoreData(array) {
+  let total = 0;
+  let min = null;
+  let max = null;
+  array.forEach((value) => {
+    total += value;
+
+    if (min == null || value < min) {
+      min = value;
+    }
+
+    if (max == null || value > max) {
+      max = value;
+    }
+  });
+
+  let mean = total / array.length;
+
+  return {
+    scores: user.scores,
+    min: min,
+    max: max,
+    mean: mean,
```

Comment:
You can use shorthand for this where the local variable has the same name as the object key:
```
return {
  scores: user.scores,
  min,
  max,
  mean
}
```